dnanexus/src/eggd_athena.sh

```
@@ -0,0 +1,86 @@
+    '''
+    [ "$summary" == 'true' ] && summary='--summary ' || unset summary
+    [ "$summary_file" == 'true' ] && summary_file='--summary_file ' || unset summary_file
+}
+
+_set_string_inputs() {
+    : '''
+    Format string inputs correctly to be passable as inputs.
+
+    If not specified will be unset, this allows them to be implicitly skipped.
+    '''
+    [ -n "$panel" ] && panel="--panel '${panel}' " || unset panel
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)